idunn/utils/result_filter.py

```diff
@@ -180,9 +184,16 @@ def check_bragi_response(query: str, bragi_response: dict) -> bool:
             }
         )
 
+    name = bragi_response["name"]
+    local_names = [
+        prop["value"]
+        for prop in bragi_response.get("properties", [])
```

Comment:
Side note: these properties are defined for POIs only.  
The issue remains for administrative regions, as Bragi returns a single name for these objects (according to the requested language). Let's ignore this for now, as it would require either to update the Bragi response, or an additional request to fetch the object from Elasticsearch.

Reply:
So the issue will kind of persist. Do you know if there is a motivation to not include these properties for admins in bragi ? It would actually be quite cheap to just add a query parameter that would disable this behavior ?

Reply:
In mimir objects, contrary to `Poi`, [`Admin`](https://github.com/CanalTP/mimirsbrunn/blob/master/libs/mimir/src/objects.rs#L516) does not contain any `properties`. Localized names are only stored in `names`. And for most geocoding use-cases, these names are not useful in the response.  

It's true that providing `properties` for all objects would be more consistent, at least as an option. But it's unclear for me whether these properties should be derived from the existing `names` field, or be added into the data model for `Admin` (and be provided at import time).